src/components/atoms/Text/Text.jsx

```diff
@@ -2,15 +2,17 @@ import PropTypes from 'prop-types';
 
 import './Text.scss';
 
-export const Text = ({ children, variant }) => {
-    return <p className={`atom__text atom__text__${variant}`}>{children}</p>;
+export const Text = ({ children, type, size }) => {
+    return <span className={`atom__text atom__text${type} atom__text${size}`}>{children}</span>;
 };
 
 Text.propTypes = {
-    children: PropTypes.node,
-    variant: PropTypes.oneOf(['primary', 'secondary'])
+    children: PropTypes.string,
```

Comment:
Here shouldn't be .isRequired? @Veryyapeee?

Reply:
good catch, it should be, we don't want to render empty component

Reply:
@Mariusz-web correct that pls

Reply:
done, i thing so